altair/examples/top_k_items.py

```diff
@@ -0,0 +1,32 @@
+"""
+Top K Items
+-------------
+This example shows how to use the window and transformation filter to display
+the Top items of a long list of items in decreasing order.
+The sorting of the x-axis is needed for vega-lite and for the example does
+not do anything since we already have a unique value (in this case a movie).
+Here we sort the top 10 highest ranking movies of IMDB.
+"""
+# category: case studies
+import altair as alt
+
+from vega_datasets import data
+movies = data.movies()
```

Comment:
```suggestion
source = data.movies.url
```
I think it would be better to use the url here. Also for the examples we try to keep some consistent variable names

Reply:
I cannot get the example to work with this suggestion. Could you replicate a good graph?